src/Sarif.ValidationTests/Sarif.ValidationTests.csproj

```diff
@@ -13,7 +13,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
 
   <ItemGroup>
-    <PackageReference Include="FluentAssertions" Version="5.4.0" />
+    <PackageReference Include="FluentAssertions" Version="5.4.1" />
```

Comment:
>5.4.1 [](start = 58, length = 5)

Because this project now has a reference to the `TestUtilities` project, and that project uses `5.4.1`.